libbeat/beat/beat.go

```diff
@@ -84,6 +85,7 @@ type BeatConfig struct {
 	Output  map[string]*ucfg.Config
 	Logging logp.Logging
 	Shipper publisher.ShipperConfig
+	Filter  []filter.FilterConfig
```

Comment:
https://github.com/elastic/beats/pull/1165 will allow to place it under beat.filter as we do we fields and tags.